application/src/main/java/org/togetherjava/tjbot/features/jshell/JShellCommand.java

```diff
@@ -48,51 +52,53 @@ public class JShellCommand extends SlashCommandAdapter {
 
     /**
      * Creates an instance of the command.
+     * 
+     * @param jshellEval the jshell evaluation instance used in the jshell command
      */
     public JShellCommand(JShellEval jshellEval) {
-        super(JSHELL_COMMAND, "JShell as a command.", CommandVisibility.GUILD);
+        super(JSHELL_COMMAND,
+                "JShell as a command, a Read–eval–print loop which allows to rapidly and simply execute java code.",
```

Comment:
"read-eval-print loop" sounds too technical imo. hard to understand as a noob. also, its kinda clear that its a command when the user reads this, as they already typed `/jshell` at that point. maybe
`Executes the given Java code and shows the output.`

Reply:
"JShell integration in discord, allows to rapidly and simply execute java code" ?

Reply:
Since this is a user facing explanation, id prefer a text that is less general and "buy this product, its good".
Maybe "Executes Java code using JShell, shows the output"

Reply:
This would be the description of /jshell eval, not the whole /jshell 🤔

Reply:
yeah. but it is its main functionality and it will be the first thing users read when they explore the command in discord by typing `/` 🤷

Reply:
I think I'll add a /jshell help instead, and keep the generic explanation, what do you think ?

Reply:
i dont think its needed. i doubt anyone would use it. i just feel like ur explanation is sorta missing the information the a regular user in our server would need to read.

Reply:
Changed, tell me what you think.